src/main/java/com/klarna/hiverunner/builder/HiveShellBase.java

```diff
@@ -200,9 +200,9 @@ public TemporaryFolder getBaseDir() {
     @Override
     public String expandVariableSubstitutes(String expression) {
         assertStarted();
-        HiveConf hiveConf = getHiveConf();
-        Preconditions.checkNotNull(hiveConf);
-        return hiveServerContainer.getVariableSubstitution().substitute(hiveConf, expression);
+        HiveConf hiveConfLocal = getHiveConf();
```

Comment:
The rule speaks about code readability. I don't think readability of these methods are a problem. It's  a rather bigger problem to call something "local" that is not local.